<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widgetSet" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelTarget">
         <property name="text">
          <string>目标ip：</string>
         </property>
         <property name="buddy">
          <cstring>leTarget</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leTarget">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labGet">
      <property name="text">
       <string>已收到信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textGet"/>
    </item>
    <item>
     <widget class="QLabel" name="labSend">
      <property name="text">
       <string>待发送信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textSend">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetBtn" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pbtnConnect">
         <property name="text">
          <string>连接</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtnSend">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
